src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write code here
+const MAX_ITEMS = 10;
+const MIN_ITEMS = 2;
+
+const element = {
+  add: (item) =>
+    item.children.length < MAX_ITEMS &&
+    item.appendChild(item.lastElementChild.cloneNode(true)),
+  remove: (item) => item.lastElementChild.remove(),
+  set: (add, item) => (add ? element.add(item) : element.remove(item)),
+};
+
+const isButton = (lengths, item) => {
+  const remove = item.className.includes('remove');
+  const append = item.className.includes('append');
+
+  switch (true) {
+    case lengths >= MAX_ITEMS && append:
+      item.disabled = true;
+      break;
+
+    case lengths === MIN_ITEMS && remove:
+      item.disabled = true;
+      break;
+
+    default:
+      item.disabled = false;
+  }
+};
+
+const container = document.querySelector('.container');
+const table = document.querySelector('tbody');
+
+container.onclick = (e) => {
```

Comment:
better to use addEventListener instead of html onClick

Reply:
Why do I need to use 'addEventListener' if I have only one event in my task? If my task had two or more events, I would use 'addEventListener'.